lib/puppet/provider/package/pip.rb

```diff
@@ -31,7 +31,11 @@ def self.instances
     packages = []
     pip_cmd = self.pip_cmd
     return [] unless pip_cmd
-    execpipe "#{pip_cmd} freeze" do |process|
+    command = ["#{pip_cmd}", 'freeze']
```

Comment:
You can rebase, we generally do rebasing and force-pushing to forks to help keep the history cleaner.